### PR TITLE
Update Timolia

### PR DESCRIPTION
**Pull request by l3nnartt**
## Type of change

- [✅] Update directory of a server

### CODE REVIEW

**Review by JakobBruening — APPROVED** [review]
Thank you!

File: minecraft_servers/timolia/manifest.json [hunk 1]
```diff
@@ -5,6 +5,10 @@
   "server_wildcards": [
     "%.timolia.de"
   ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
```
Comment (scrummer) [inline minecraft_servers/timolia/manifest.json:11]: @l3nnartt Obviously I was a bit to slow but doesn't Timolia support more than these two languages?

Comment (l3nnartt) [reply minecraft_servers/timolia/manifest.json:11]: Yes (de/en/es/ru/ua), but to be fair I have to say that the translations are not complete.
I don't know if that matters?!

Comment (scrummer) [reply minecraft_servers/timolia/manifest.json:11]: Okay, then just leave it like that. thx!

**Pull request merged**